.gitignore

```diff
@@ -1,4 +1,6 @@
 __pycache__/
 .env
 data/
-backend/src/source_list.json
\ No newline at end of file
+backend/src/source_list.json
+.ruff_cache
+backend/src/secret.json
```

Comment:
What is this used for?

Reply:
It has the API keys required for using the newer models on Google's VertexAI platform. I'll add an option to use these models in the next PR.